src/main/java/book/store/model/Role.java

```diff
@@ -0,0 +1,33 @@
+package book.store.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Data;
+import org.springframework.security.core.GrantedAuthority;
+
+@Data
```

Comment:
avoid using @ Data on JPA entities, it has side effects (toString, equalsAndHashcode specifically)